web/client/components/map/openlayers/Map.jsx

```diff
@@ -162,6 +162,11 @@ class OpenlayersMap extends React.Component {
         this.map.enableEventListener = (event) => {
             delete this.map.disabledListeners[event];
         };
+        this.map.getViewport().addEventListener('mouseout', () => {
+            if (this.props.mousePosition) {
+                setTimeout(() => this.props.onMouseOut(), 150);
```

Comment:
There is any reason to postpone this event by 150 ms? If no, please remove the delay

Reply:
We need this 150ms delay because  the mouseMoveEvent has delay 100ms:
![image](https://user-images.githubusercontent.com/61145641/79864347-02bcda00-83e2-11ea-8d07-cf8b9b9cda2f.png)